### DISCUSSION TITLE
Categories... ?

### DISCUSSION THREAD

**Discussion by cdanca** [PodcastGenerator/PodcastGenerator#683] [Q&A] [answered]
Hello, GH friends, I am new at using Podcast Generator and when I am about to upload a program, there is an option for adding "categories" to our program, but I don´t find a way to set up those categories, so it only appear the word: "Uncategorized", how I can set up categories to chose from?

Thanks in advance.

Daniel Cajiga
![Uncategorized](https://user-images.githubusercontent.com/47028440/180308089-96bd59b4-ea62-4d65-8ebf-163b1b1bb6cc.png)

**Comment by charabaruk** [comment, marked as answer]
Hi Daniel,

Under the Episodes menu in the admin header bar, there's an item titled "Manage categories". It is the third item in the menu. On that page you are able to create or delete categories.
